modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy

```diff
@@ -27,45 +28,116 @@
 {template .select autoescape="deprecated-contextual"}
```

Comment:
Please run check_sf on this file before sending. Here's the output: 

dynamic-data-mapping-type-select ➤ check_sf src/main/resources/META-INF/resources/**/*.soy                                                                                                     git:pr-796
File: src/main/resources/META-INF/resources/select.soy
    Line 41: Sort attribute values: input-group-container form-builder-select-field
    Line 55: Sort attribute values: select-field-trigger form-control
    Line 55: Sort attributes: href dir